apis/v1alpha1/generator.yaml

```
@@ -153,6 +153,9 @@ resources:
             operation: ReadOne
     tags:
       ignore: true
+    hooks:
+      sdk_delete_pre_build_request:
+        template_path: hooks/layerversion/sdk_delete_pre_build_request.go.tpl
```

nit: We use `snake_casing` for resource names. 
```suggestion
template_path: hooks/layer_version/sdk_delete_pre_build_request.go.tpl
```

---

Hey @RedbackThomson, 
Thanks so much for bringing it, I have changed it to snake_casing